modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,329 @@
+
+. Enable the pods in the individual user namespace to access the `/dev/fuse` and `/dev/net/tun` devices on the host node.  
+
+.. Create a machine config similar to the following example:
++
+[source,terminal]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 99-worker-allowed-devices
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - path: /etc/crio/crio.conf.d/99-allowed-devices.conf
+        mode: 0640
+        overwrite: true
+        contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KYWxsb3dlZF9kZXZpY2VzID0gWyIvZGV2L2Z1c2UiLCAiL2Rldi9uZXQvdHVuIl0K <1>
+----
+<1> Specifies the Base64-encoded command: `allowed_devices = ["/dev/fuse", "/dev/net/tun"]`.
+////
+
+.Procedure
+
+. Enable the use of Linux user namespaces by creating a pod configured to run with a `restricted` profile and with the `hostUsers` parameter set to `false`:
+
+.. Create a YAML file similar to the following:
++
+.Example pod specification
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: userns
+
+# ...
+
+spec:
+  containers:
+  - name: userns
+    image: registry.access.redhat.com/ubi9
+    command: ["sleep", "1000"]
+    securityContext:
+      capabilities:
+        drop: ["ALL"]
+      allowPrivilegeEscalation: false <1>
+      runAsNonRoot: true <2>
+      seccompProfile:
+        type: RuntimeDefault
+      runAsUser: 1024 <3>
+      runAsGroup: 1024 <4>
+  hostUsers: false <5>
+
+# ...
+----
+<1> Specifies that a pod cannot request privilege escalation.
+<2> Use a non-root user inside the container.
+<3> Specifies the user ID (UID) the container is run with.
+<4> Specifies which primary group ID (GID) the containers is run with.
+<5> Allows the pod to run in a user namespace. If `true`, the pod runs in the host user namespace. If `false`, the pod runs in a new user namespace that is created for the pod. The default is `true`.
+
+.. Create the pod by running the following command:
++
+----
+$ oc create -f <file_name>.yaml
+----
+
+. Edit the default user ID (UID) and group ID (GID) range of the {product-title} namespace where your pod is deployed by running the following command:
++
+[source,terminal]
+----
+$ oc edit ns/<namespace_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+piVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/requester: system:admin
+    openshift.io/sa.scc.mcs: s0:c27,c24
+    openshift.io/sa.scc.supplemental-groups: 1024/10000 <1>
+    openshift.io/sa.scc.uid-range: 1024/10000 <2>
+# ...
+name: userns
+# ...
+----
+<1> Edit the default GID to match the value you specified in the pod spec. The range for a Lunix user namespace must be lower than 65,535. The dafualt is `1000000000/10000`.
```

Comment:
default*

Reply:
Sorry! I usually do a spell check pass. Must have missed it this time.